samples/sensor/lis3dh_test/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_STDOUT_CONSOLE=y
```

Comment:
check the other merged PRs for an example for specifying which configs are already included in the board config but just added here to be sure, and which ones aren't

Reply:
Done

Reply:
[Commit](https://github.com/abelsensors/zephyr/commit/a583d21a309f806e0aa53a75107c493e0a623df2)
Is this correct?

Reply:
Yes, maybe also make a section for debugging/logging stuff if you're going to separate that from the other two, but this is what I meant